cmd/oras/internal/display/status/print.go

```diff
@@ -41,11 +43,14 @@ func NewPrinter(out io.Writer) *Printer {
 }
 
 // Println prints objects concurrent-safely with newline.
-func (p *Printer) Println(a ...any) error {
+func (p *Printer) Println(a ...any) {
 	p.lock.Lock()
 	defer p.lock.Unlock()
 	_, err := fmt.Fprintln(p.out, a...)
-	return err
+	if err != nil && p.errored == nil {
```

Comment:
Say we have an error `A`, which auto resolved and we have another error `B/

Should `A` and `B` be output to stderr? If so then we need to change it to
```suggestion
	if err != nil {
```

Reply:
If `&& p.errored == nil` is used then `errored` will always be wrapped `A` but not changed to `B`

Reply:
It be very unusual to have A not the same as B and probably not worth distinguishing. I'm okay with last error wins, but I'd like to report it only once and it is easier to just report the first.

Reply:
To me the most reasonable solution is to report all the errors.

Reply:
Done

Reply:
If we agree to report all the errors, should the error be returned directly via `printer.Println`?

Reply:
like
```golang
// Println prints objects concurrent-safely with newline.
func (p *Printer) Println(a ...any) error {
	p.lock.Lock()
	defer p.lock.Unlock()
	_, err := fmt.Fprintln(p.out, a...)
	if err != nil  {
		err = fmt.Errorf("display output error: %w", err)
		_, _ = fmt.Fprint(os.Stderr, err)
	}
        return err
}
```

Reply:
so `errored` is no longer needed.

Reply:
Sounds good to me